docs/plugins/virtual_environment_docs/3-version-management.md

```diff
@@ -0,0 +1,44 @@
+# Version management  
+
+This guide covers the methods of updating your version string everywhere.
+
+## This guide covers:  
+* [Using git tags](#using-git-tags)
+* [Using a local script to edit files](#using-a-local-script-to-edit-files)  
+* [Manually](#manually)
+
+Your goal is to make sure that you bump your version string everywhere it may appear, in unison, prior to publishing your package.  A version number can be in `init.py`, `setup.cfg`, etc.
+
+In increasing order of work, but decreasing order of magic, the methods of bumping your version string are listed below. 
+
+## Using git tags:  
+You can use [setuptools_scm](https://github.com/pypa/setuptools_scm) to automatically generate version numbers for your package based on tagged commits.
+
+   `# configure in pyproject.toml, then…`  
```

Comment:
These two lines should be a codeblock with triple backticks.

Reply:
Done.